### DISCUSSION TITLE
Help with F2010C5-CMIP6-HR compset

### DISCUSSION THREAD

**Discussion by anasena** [E3SM-Project/E3SM#4630] [E3SM model help]
Hi!
I am trying to run a simulation using the F2010C5-CMIP6-HR on a ne120np4_oRRS18to6v3 grid.
I am running into two main errors:

1)	With the new release, on Compy, the model complains about the lack of inputdata for CAM. I have attached some of the messages it spells out when building.

2)	In tests with a previous E3SM release on Cori, I kept running into an error message about the size of the forcing files:
“0:  ERROR: (shr_dmodel_readstrm) ERROR in data sizes
5411: (shr_dmodel_readstrm) ERROR in data sizes      360     180      12 1036800”
The forcing files are  monthly, 1o grid SST/seaice. The same forcing works for the SSP85 period (compset: SSP85_CAM5%CMIP6-HR_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV).
I have also tried with a forcing file on a 0.25o grid, and also with the grid ne30np4, with no luck.
In this last case, the error states: "negative layer thickness. timestep or remap time too large"

Any input would be appreciated!
Best,
Ana

[error_message_compy_F2010HR.txt](https://github.com/E3SM-Project/E3SM/files/7436318/error_message_compy_F2010HR.txt)

**Comment by jonbob** [comment]
@anasena - I don't think that compset is supported in the new release. Did you use a different compset? I looked in the files referenced by the text error file you attached and couldn't find the case you were working on

**Comment by mt5555** [comment]
There is a little bit of documentation on that compset here: 
https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/925631509/F2010C5-CMIP6-HR
It is for the E3SM V1 release, so as Jon said, may not work in newer versions of the code.  

If you need to run that compset, you could try the "maint-1.0" branch.

**Reply by anasena** [reply]
Thank you both! I think that may be part of the problem: I accidentally cloned maint-1.0 instead of the new version.
These errors appear on that version.
I will try with a new compset, in the newly released version.
